File: app/src/main/java/com/visiontutor/finalproject/utils/Tutor.java
package com.visiontutor.finalproject.utils;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Tutor {

    @SerializedName("image")
    @Expose
    private String image;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("lastname")
    @Expose
    private String lastname;
    @SerializedName("about")
    @Expose
    private String about;
    @SerializedName("gender")
    @Expose
    private String gender;
    @SerializedName("yob")
    @Expose
    private int yob;
    @SerializedName("tutorid")
    @Expose
    private String tutorid;
    @SerializedName("medium")
    @Expose
    private String medium;
    @SerializedName("timeteach")
    @Expose
    private String timeteach;
    @SerializedName("area")
    @Expose
    private String area;
    @SerializedName("city")
    @Expose
    private String city;

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getName() {
        return name + " " + lastname;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getLastname() {
        return lastname;
    }

    public void setLastname(String lastname) {
        this.lastname = lastname;
    }

    public String getAbout() {
        return about;
    }

    public void setAbout(String about) {
        this.about = about;
    }

    public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

    public int getYob() {
        return yob;
    }

    public void setYob(int yob) {
        this.yob = yob;
    }

    public String getTutorid() {
        return tutorid;
    }

    public void setTutorid(String tutorid) {
        this.tutorid = tutorid;
    }

    public String getMedium() {
        return medium;
    }

    public void setMedium(String medium) {
        this.medium = medium;
    }

    public String getTimeteach() {
        return timeteach;
    }

    public void setTimeteach(String timeteach) {
        this.timeteach = timeteach;
    }

    public String getArea() {
        return area;
    }

    public void setArea(String area) {
        this.area = area;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

}

File: app/src/main/java/com/visiontutor/app/utils/Tutor.kt
package com.visiontutor.app.utils

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Tutor {

    @SerializedName("id")
    @Expose
    var id: Int = 0

    @SerializedName("image")
    @Expose
    var image: String? = null

    @SerializedName("name")
    @Expose
    var name: String? = null
        get() = "$field $lastname"

    @SerializedName("lastname")
    @Expose
    var lastname: String? = null

    @SerializedName("about")
    @Expose
    var about: String? = null

    @SerializedName("gender")
    @Expose
    var gender: String? = null

    @SerializedName("yob")
    @Expose
    private var yob: Int = 0

    @SerializedName("tutorid")
    @Expose
    var tutorid: String? = null

    @SerializedName("medium")
    @Expose
    var medium: String? = null

    @SerializedName("timeteach")
    @Expose
    var timeteach: String? = null

    @SerializedName("area")
    @Expose
    var area: String? = null

    @SerializedName("city")
    @Expose
    var city: String? = null

    fun getYob(): String {
        return yob.toString()
    }

    fun setYob(yob: Int) {
        this.yob = yob
    }

}